example/javascriptlib/basic/1-simple/build.mill

```diff
@@ -1,27 +1,58 @@
 package build
+
 import mill._, javascriptlib._
 
-object foo extends TypeScriptModule {
-  object bar extends TypeScriptModule {
-    def npmDeps = Seq("immutable@4.3.7")
+trait SimpleMod extends JestModule {
+  override def npmInstall = Task {
+    os.call((
+      "npm",
+      "install",
+      "--save-dev",
+      "@types/node@22.7.8",
+      "@types/jest@^29.5.14",
+      "@babel/core@^7.26.0",
+      "@babel/preset-env@^7.26.0",
+      "typescript@5.6.3",
+      "jest@^29.7.0",
+      "ts-jest@^29.2.5",
+      "ts-node@^10.9.2",
+      "babel-jest@^29.7.0",
+      "esbuild@0.24.0",
+      "immutable@4.3.7",
+      transitiveNpmDeps()
```

Comment:
This list is very long. Where does it come from? Are all of them needed? How can a developer know to come up with this list themselves?

Reply:
ok, will move out the common packages JestModule depends on and leave "immutable" in SimpleMod since its project specific.